packages/lesswrong/components/ea-forum/EACollectionCard.tsx

```diff
@@ -47,13 +47,22 @@ const EACollectionCard = ({collection}: {collection: CollectionsBestOfFragment})
   const {title, author, imageId} = getCardDetails(collection);
   const href = collectionGetPageUrl(collection);
 
-  const {EASequenceOrCollectionCard, CollectionsHoverOver} = Components;
+  const TitleWrapper = useCallback(({children}: {children: ReactNode}) => {
+    const {CollectionsTooltip} = Components;
```

Comment:
If we descructed it outside then it would need to become a dependency of the call to `useCallback`. Also, I like only having it available in the scope where it's actually used.